modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/notification/template/mgt/OrganizationNotificationOrgTemplatesTestCase.java

```diff
@@ -0,0 +1,527 @@
+                add("internal_template_mgt_update");
+                add("internal_template_mgt_delete");
+            }};
+    private static final String API_RESOURCE_ORGANIZATION_TYPE = "ORGANIZATION";
+    private static final List<String> NOTIFICATION_TEMPLATE_MANAGEMENT_API_ORG_SCOPES =
+            new ArrayList<String>() {{
+                add("internal_org_template_mgt_view");
+                add("internal_org_template_mgt_create");
+                add("internal_org_template_mgt_update");
+                add("internal_org_template_mgt_delete");
+            }};
+    private static final String SUBJECT = "subject";
+    private static final String BODY = "body";
+    private static final String FOOTER = "footer";
+    private static final String CODE = "code";
+    private static final String NOT_EXIST_ERROR_CODE = "NTM-65004";
+
+    private static final String L1_SUB_ORG1_NAME = "L1_Sub_Org1";
+    private static final String L2_SUB_ORG1_NAME = "L2_Sub_Org1";
+    private static final String TEST_LOCALE = "fr_FR";
+    private static final String TEST_CONTENT_TYPE = "text/html";
+    private static final String TEST_DATA_SUBJECT_1 = "Test email subject 1";
+    private static final String TEST_DATA_BODY_1 = "Test email body 1";
+    private static final String TEST_DATA_FOOTER_1 = "Test email footer 1";
+    private static final String TEST_DATA_SUBJECT_2 = "Test email subject 2";
+    private static final String TEST_DATA_BODY_2 = "Test email body 2";
+    private static final String TEST_DATA_FOOTER_2 = "Test email footer 2";
+    private static final String TEST_DATA_SUBJECT_3 = "Test email subject 3";
+    private static final String TEST_DATA_BODY_3 = "Test email body 3";
+    private static final String TEST_DATA_FOOTER_3 = "Test email footer 3";
+    private static final String TEST_DATA_SUBJECT_4 = "Test email subject 4";
+    private static final String TEST_DATA_BODY_4 = "Test email body 4";
+    private static final String TEST_DATA_FOOTER_4 = "Test email footer 4";
+    private static final String TEST_DATA_SUBJECT_5 = "Test email subject 5";
+    private static final String TEST_DATA_BODY_5 = "Test email body 5";
+    private static final String TEST_DATA_FOOTER_5 = "Test email footer 5";
+    private static final String TEST_DATA_SUBJECT_6 = "Test email subject 6";
+    private static final String TEST_DATA_BODY_6 = "Test email body 6";
+    private static final String TEST_DATA_FOOTER_6 = "Test email footer 6";
+    private static final String TEST_EMAIL_TEMPLATE_ID = "aWRsZUFjY291bnRSZW1pbmRlcg";
+
+    private OrgMgtRestClient orgMgtRestClient;
+    private NotificationTemplatesRestClient notificationTemplatesRestClient;
+
+    private String level1Org1Id;
+    private String level2Org1Id;
+    private String switchedM2MTokenLevel1Org1;
+    private String switchedM2MTokenLevel2Org1;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+
+        super.init();
```

Comment:
run this for two user modes, super tenant and another tenant

Reply:
Added with https://github.com/wso2/product-is/pull/22807/commits/17b8905a1cdfdd60039d5d94ff4bcb49932e7f33